scripts/analyze_embedding_efficiency.py

```
@@ -44,281 +46,327 @@
-    
+
     def __init__(self):
         """Initialize the analyzer."""
         self.langfuse_client = get_langfuse_client()
         if not self.langfuse_client.enabled:
-            raise RuntimeError("Langfuse is not enabled. Please configure Langfuse settings.")
-    
-    def analyze_model_performance(self, hours_back: int = 24) -> List[ModelPerformanceMetrics]:
+            raise RuntimeError(
+                "Langfuse is not enabled. Please configure Langfuse settings."
+            )
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)